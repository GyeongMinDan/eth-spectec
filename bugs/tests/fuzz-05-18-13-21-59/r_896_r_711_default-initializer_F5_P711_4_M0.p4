   control C(out bit<32> x)() {
    tuple<_> stack;
    apply {
   }
  }
