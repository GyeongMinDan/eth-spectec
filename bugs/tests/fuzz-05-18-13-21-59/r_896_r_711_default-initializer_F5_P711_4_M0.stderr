likely_hits/r_896_r_711_default-initializer_F5_P711_4_M0.p4(1): [--Wwarn=unused] warning: Control C is not used; removing
   control C(out bit<32> x)() {
           ^
[--Wwarn=missing] warning: Program does not contain a `main' module
